calculator.js

```diff
@@ -5,24 +5,70 @@ const exampleAdditionInput = {
   operation: 'add',
 }
 
+const checkNum = function (num) {
+  if (isNaN(num)) {
+    console.log(`${num} is not a number.  Try again.`)
+    return false
+  } else {
+    return true
+  };
+}
+
+const printResult = function (operator, num1, num2, result) {
+  return console.log(`${ num1 } ${operator} ${ num2 } = ${ result }`);
+}
+
 const calculateUserInput = function (error, promptInput) {
   console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
 
+  if (!checkNum(promptInput.num1) || !checkNum(promptInput.num2)) {
+    return
+  };
+
+  const num1 = parseInt(promptInput.num1);
+  const num2 = parseInt(promptInput.num2);
+  const operation = promptInput.operation.toLowerCase();
+  let result = 0;
+
+  switch (operation) {
+    case 'add':
+    case '+':
+      result = num1 + num2;
```

Comment:
Consider encapsulating the functionality of each operation in it's own function.